weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewerSection.tsx

```diff
@@ -126,19 +129,29 @@ const ObjectViewerSectionNonEmpty = ({
     },
     [apiRef]
   );
+  const getGroupIds = useCallback(() => {
+    const rowIds = apiRef.current.getAllRowIds();
+    rowIds.filter(rowId => {
```

Comment:
This doesn't look right, you're not doing anything with the return value.

Reply:
shocking code quality last week 🤦🏻